test/core/client_channel/lb_policy/lb_policy_test_lib.h

```diff
@@ -658,11 +659,16 @@ class LoadBalancingPolicyTest : public ::testing::Test {
   // Gets num_picks complete picks from picker and returns the resulting
   // list of addresses, or nullopt if a non-complete pick was returned.
   absl::optional<std::vector<std::string>> GetCompletePicks(
-      LoadBalancingPolicy::SubchannelPicker* picker, size_t num_picks,
-      SourceLocation location = SourceLocation()) {
+      LoadBalancingPolicy::SubchannelPicker* picker,
+      const std::map<UniqueTypeName, std::string> call_attributes = {},
+      size_t num_picks = 3, SourceLocation location = SourceLocation()) {
```

Comment:
I think `num_picks` should not have a default value here, because there isn't really a default number of picks that will be appropriate in most circumstances.  The caller should be required to pass this parameter.

Reply:
Instead of dropping the default for `call_attributes`, just reorder the parameters so that `num_picks` is before `call_attributes`.

I've done this in https://github.com/eugeneo/grpc/pull/1.